scripts/integrity.js

```diff
@@ -75,4 +77,4 @@ for (var i = 0; i < config.fontawesome.length; i++) {
     }
 }
 
-fs.writeFileSync(configFile, yaml.dump(config));
+fs.writeFileSync(configFile, yaml.dump(config, { lineWidth: 110 }));
```

Comment:
I read over everything else and it looks good. I just wasn't familiar with the `lineWidth` option. I'll google. :P